src/app/page.tsx

```diff
@@ -2,7 +2,7 @@ import Image from "next/image";
 
 export default function Home() {
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
+    <main className="min-h-screen flex-col items-center justify-between p-24">
```

Comment:
Consider adding a comment to explain the reason behind removing the `flex` class. This will help future contributors understand the change better.